src/KeyVault/KeyVault/Models/Secret/KeyVaultSecretUri.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace Microsoft.Azure.Commands.KeyVault.Models.Secret
+{
+    /// <summary>
+    /// A data class to hold components of a KeyVault Secret URI: VaultName, SecretName, and SecretVersion.
+    /// </summary>
+    public class SecretUriComponents
```

Comment:
can we let the access level be internal? seems it's no need that be exposed to customer.
```suggestion
    internal class SecretUriComponents
```

Reply:
Update: can't change the accessibility identifier to `internal` because the helper method has package access (`protected`). This causes inconsistent accessibility.

Reply:
Changed it back to `public`

Reply:
Can we change it to protected?

Reply:
Tried doing that, Got this error:

CS1527: Elements defined in a namespace cannot be explicitly declared as private, protected,
protected internal, or private protected